File: conformance/bundles/19-emphasis-parameter/mention.steps.go
// Go sibling of mention.steps.ts (bundle 19-emphasis-parameter).
package fixture

import "github.com/varar-dev/varar/go/varar"

func Register(s *varar.Steps[varar.Value]) {
	s.Stimulus("I mention {emph}", func(state varar.Value, args []varar.Value) (*varar.Value, error) {
		return nil, nil
	})
}

func State() varar.Value {
	return varar.NullValue
}

File: conformance/bundles/19-emphasis-parameter/mention.steps.cs
// C# sibling of mention.steps.ts (bundle 19-emphasis-parameter). {emph} is a
// built-in parameter type (Markdown emphasis); matching is what conformance pins.
using Varar;
using Varar.Core;

namespace Varar.Corpus.B19;

public static class MentionSteps
{
    public static void Register(Steps s)
    {
        s.Stimulus("I mention {emph}", (state, text) => null);
    }

    public static Value State() => Value.Null;
}
